ports/fp16/vcpkg.json

```
@@ -0,0 +1,9 @@
+{
+  "name": "fp16",
+  "version-string": "2021-02-21",
```

```suggestion
  "version-date": "2021-02-21",
```

We prefer to use `version-date` in this case. Please see https://github.com/microsoft/vcpkg/blob/master/docs/specifications/versioning.md#22-package-versions

---

Updated in 4cdaa9d. Thanks for the point!